### PR TITLE
yelp tools

### PR DESCRIPTION
**Pull request by Toasterson**
Adding prerequisite for mate yelp-tools

### CODE REVIEW

File: components/gnome/yelp-tools/Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+COMPONENT_BUILD_ENV += CFLAGS="$(CFLAGS)"
+
+# common targets
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+test:		$(NO_TESTS)
+
+BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
```
Comment (pyhalov) [inline components/gnome/yelp-tools/Makefile]: BUILD_PKG_DEPENDENCIES and depend.mk are obsolete

**Comment by pyhalov** [comment]
Please, move it to components/desktop/gnome3/yelp-tools or at least components/desktop/yelp-tools

File: components/gnome/yelp-tools/Makefile [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+# common targets
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+test:		$(NO_TESTS)
+
+REQUIRED_PACKAGES += gnome/help-viewer/yelp
+REQUIRED_PACKAGES += text/itstool
```
Comment (pyhalov) [inline components/gnome/yelp-tools/Makefile]: Add REQUIRED_PACKAGES += library/libxslt ?

**Comment by pyhalov** [comment]
Again you forgot to fix path to shared-macros.mk

File: components/desktop/gnome3/yelp-tools/Makefile [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+# common targets
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+test:		$(NO_TESTS)
+
+REQUIRED_PACKAGES += gnome/help-viewer/yelp
+REQUIRED_PACKAGES += text/itstool
```
Comment (pyhalov) [inline components/desktop/gnome3/yelp-tools/Makefile:57]: Also library/libxml2, library/libxslt , yelp-xsl and text/gawk

File: components/desktop/gnome3/yelp-tools/Makefile [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+REQUIRED_PACKAGES += library/libxml2
+REQUIRED_PACKAGES += library/libxslt
+REQUIRED_PACKAGES += yelp-xsl
```
Comment (pyhalov) [inline components/desktop/gnome3/yelp-tools/Makefile]: Correct FMRI is text/yelp-xsl

**Comment by pyhalov** [comment]
Merge commits. Check commit message (last time I failed to notice that it included commit messages from all intermediate commits).

**Comment by Toasterson** [comment]
Dont you have a button merge and squash in github? Arcording to some docs that button should exist

**Comment by pyhalov** [comment]
This creates different history...

**Pull request merged**